extensions/cornerstone/src/utils/measurementServiceMappings/LivewireContour.ts

```diff
@@ -142,19 +147,26 @@ function getDisplayText(annotation, displaySet, customizationService) {
   const frameText = displaySet.isMultiFrame ? ` F: ${frameNumber}` : '';
```

Comment:
This bit of code for the series text seems like it is repeated over and over again.  Can you add it to a parent class?

Reply:
I initially planned to create a parent class, but things got hectic. Let's do it in the next release.https://github.com/OHIF/Viewers/issues/4458